### DISCUSSION TITLE
Using load_var to get secrets from AWS

### DISCUSSION THREAD

**Discussion by SukhHeer** [concourse/concourse#8735] [Help & Support] [answered]
I have an existing task which sets the param up as the below, this same task then echo's ENV_PASSWORDS which confirms that the secrets have been retrieved and contain the expected values.

ENV_PASSWORDS: ((automation/((appName))/((autoDeployEnvironment))))

I am trying to use load_var to get the secrets but this is not working. load_var has been defined just prior to the secrets task that gets run.

ENV_PASSWORDS: ((automation/((appName))/((.:env))))

The echo of ENV_PASSWORDS shows the value as ((automation/Test-App/stage))

So the values are correct thats the location of the secrets in AWS but when using load_var I just get the value back rather then the secrets at that location. 

Is it possible to retrieve secrets via load_var? 

I have tried lots of different variations with no success 

Any ideas?

**Comment by evanchaoli** [comment, marked as answer]
Looks like you are trying to build a var name using `load_var`, which is not supported.

**Reply by SukhHeer** [reply]
Thanks for the reply, I'll have a think on how to move around this in that case
